projects/example-app/src/app/books/actions/book.actions.ts

```diff
@@ -1,5 +1,5 @@
+import { Book } from '@example-app/books/models';
```

Comment:
Don't re-order the imports. I believe the style guide says order them by name, but they are fine as-is. Its preference, but we order them this way with a line break in between each set.

1. Framework imports (@angular/core)
2. Third-party imports (@angular/material, @ngrx/store)
3. Local imports (@example-app)

Reply:
Ok, no problem @brandonroberts 
That was WebStorm's 'optimize imports' formatting

Reply:
@brandonroberts I reordered the imports per preference you mentioned previously. Let me know if we are clear to merge. Thanks